test/libsolidity/SemanticTest.cpp

```diff
@@ -304,6 +330,17 @@ TestCase::TestResult SemanticTest::runTest(
 			test.setRawBytes(move(output));
 			test.setContractABI(m_compiler.contractABI(m_compiler.lastContractName(m_sources.mainSourceFile)));
 		}
+
+		if (test.call().actualSideEffects.empty())
```

Comment:
Can it actually be non-empty at this point for a legitimate reason? Wouldn't that be some unwanted leftover from a previous run or something?